pom.xml

```diff
@@ -33,107 +33,8 @@
   </licenses>
 
   <modules>
-    <!-- super pom -->
-    <module>super/pom.xml</module>
```

Comment:
Is this module no longer needed here?

Reply:
That module doesn't produce artifacts, it's referenced by other modules as super-pom. It shouldn't be built on its own.